src/privileged/FirefoxMonitor.css

```diff
@@ -0,0 +1,85 @@
+#fxmonitor-notification .popup-notification-body {
+  font-size: 110%;
+}
+
+#fxmonitor-notification popupnotificationcontent {
+  margin-top: 0;
+}
+
+#fxmonitor-notification .popup-notification-body > hbox:first-child {
+  display: none;
+}
+
+#fxmonitor-notification .popup-notification-icon,
+.fxmonitor-icon {
+  list-style-image: url("chrome://browser/skin/warning.svg");
+  fill: rgba(12, 12, 13, 0.6); /* Gray 90 a60 */
```

Comment:
This looks really bad in dark theme. I understand that warning.svg doesn't support context-fill-opacity, which is actually another reason to just ship your own...